src/pkg/projects/Microsoft.NETCore.App/src/Microsoft.NETCore.App.depproj

```diff
@@ -263,12 +263,16 @@
       <_crossGenArgs Include="-platform_assemblies_paths $(_crossgenPlatformAssemblies)" />
       <_crossGenArgs Include="-JITPath $(_jitPath)" />
     </ItemGroup>
+    <PropertyGroup>
+      <_partialNgen Condition="'$(OS)' == 'Windows_NT'">1</_partialNgen>
+      <_partialNgen Condition="'$(OS)' != 'Windows_NT'">0</_partialNgen>
```

Comment:
Can you share why we're only enabling partial NGEN for Windows and not UNIX?

Reply:
On Windows, we will have IBC data, while on linux we will not. When we have ibc data, we want to partial crossgen the entire framework to get the size benefits (so the assemblies with ibc data will be partial ngened, where as the ones without will be IL). On Linux, where we won't have ibc data yet, we don't want to stop crossgening the entire framework, so we don't want to use partial ngen.

Reply:
Gotcha.  So, this is just a point-in-time thing?